settings.py

```diff
@@ -70,7 +70,7 @@
 ES_DEFAULT_NUM_REPLICAS = 0
 
 SITE_URL = os.environ.get('OLYMPIA_SITE_URL') or 'http://localhost:8000'
-SERVICES_DOMAIN = 'localhost:8000'
+SERVICES_DOMAIN = SITE_URL.replace('http://', '').replace('https://', '')
```

Comment:
Nitpick, but can't we just do a regex replace (`/https?:\/\//`) instead of two replace calls? Seems neater to me.

But I dunno if that's more annoying in Python, I forget 😄